pinot-common/pom.xml

```diff
@@ -36,6 +36,12 @@
   </properties>
 
   <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
```

Comment:
what was this for?

Reply:
filtering (see the javadoc of `PinotVersion`) is what allows maven to understand that it should replace `${project.version}` with the actual version